src/chainlit/message.py

```diff
@@ -141,6 +143,7 @@ def __init__(
         prompt: str = None,
         llm_settings: LLMSettings = None,
         language: str = None,
+        parent_id: Any = None,
```

Comment:
could be `Union[str, int]`

Reply:
What if the user as a custom database client that use another type as an id? (e.g. UUID, MongoDB's ObjectId, etc.)

Reply:
ok, let's assume ids are ints for now